Sources/MongoSwift/MongoCollection+Read.swift

```diff
@@ -278,67 +278,67 @@ public enum CursorType {
 /// Options to use when executing a `find` command on a `MongoCollection`.
 public struct FindOptions: Codable {
     /// Get partial results from a mongos if some shards are down (instead of throwing an error).
-    public let allowPartialResults: Bool?
+    public var allowPartialResults: Bool?
 
     /// The number of documents to return per batch.
-    public let batchSize: Int32?
+    public var batchSize: Int32?
 
     /// Specifies a collation.
-    public let collation: Document?
+    public var collation: Document?
 
     /// Attaches a comment to the query.
-    public let comment: String?
+    public var comment: String?
 
     /// If a `CursorType` is provided, indicates whether it is `.tailable` or .`tailableAwait`.
-    private let tailable: Bool?
```

Comment:
I think we can also leave `tailable` and `await` as `let` since we don't want to be modifying them directly but only through the new cursor type property.

Reply:
If we declare them with `let`, how would we modify them as needed in my example above?